molgenis-metadata-manager/src/main/frontend/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "frontend",
+  "version": "1.0.0",
+  "description": "MOLGENIS Metadata manager",
+  "author": "Mark-de-Haan <markdehaan90@gmail.com>",
+  "private": true,
+  "scripts": {
+    "dev": "node build/dev-server.js",
+    "build": "node build/build.js",
+    "test": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
+    "test:watch": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --watch",
+    "test:debug": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js -i --watch",
+    "lint": "eslint --ext .js,.vue src test/unit/specs",
+    "lint:fix": "eslint --ext .js,.vue src test/unit/specs"
+  },
+  "dependencies": {
+    "@molgenis/molgenis-i18n-js": "^0.0.5",
+    "@molgenis/molgenis-api-client": "^1.0.0",
+    "bootstrap": "4.0.0-alpha.6",
+    "flow": "^0.2.3",
+    "flow-babel-webpack-plugin": "^1.0.1",
+    "font-awesome": "^4.7.0",
+    "v-toaster": "^1.0.2",
+    "vue": "^2.2.6",
+    "vue-multiselect": "next",
```

Comment:
Normal version numbers or not yet compatible with Vue2, this is the only option we get.